Makefile

```diff
@@ -21,9 +21,11 @@
 
 ###################################################################################
 
+# See https://tech.davis-hansson.com/p/make/ for more information
+# about these initial lines.
 MAKEFLAGS += --warn-undefined-variables --no-builtin-rules
 SHELL := /usr/bin/env bash
-.SHELLFLAGS := -uo pipefail -c
+.SHELLFLAGS := -eu -o pipefail -c
```

Comment:
I wish I could remember why we don't have errexit set in the cert-manager makefile. I feel like it broke something, but don't have time to dig into it 🙃 

If it works here though, all the better!